.packit.sh

```diff
@@ -32,3 +32,7 @@ sed -i "s/^Release:.*/Release: ${PACKIT_RPMSPEC_RELEASE}%{?dist}/" ${SPEC_FILE}
 
 # Update Source tarball name in spec
 sed -i "s/^Source0:.*.tar.gz/Source0: %{name}-${HEAD_VERSION}.tar.gz/" ${SPEC_FILE}
+
+# Update build additional rpms in spec
+sed -i 's/\(enable_qm_dropin_img_tempdir \).*/\11/' ${SPEC_FILE}
```

Comment:
Question: Is it now auto enabling iot img_tempdir and ROS subpackages in QM now? It's ok from my side (at least in upstream).

Reply:
It was by default. I assume you wanted it in. 
I will add only kvm. once i will solve the failures